Jenkinsfile

```diff
@@ -132,7 +132,7 @@ pipeline {
                 }
             }
             steps{
-                withMaven(maven: 'Maven 3.5.4', jdk: 'jdk8-latest', globalMavenSettingsConfig: 'default-global-settings', mavenSettingsConfig: 'codice-maven-settings', mavenOpts: '${LINUX_MVN_RANDOM}') {
+                withMaven(maven: 'maven-latest', jdk: 'jdk8-latest', globalMavenSettingsConfig: 'default-global-settings', mavenSettingsConfig: 'codice-maven-settings', mavenOpts: '${LINUX_MVN_RANDOM}') {
```

Comment:
Only worry I have with this PR, is how this might impact deploy. Not really sure we will know, until we merge this PR though.

Reply:
Yes, definitely a possibility. Agree  it is more about the deploy plugin version. Doesn't look like we specify it here so it would grab the latest, which might work by now. Not sure.

Reply:
tested the deploy phase and it worked